### PR TITLE
Describe two new `rel` link types.

### PR DESCRIPTION
**Pull request by mikewest**
This PR adds descriptions for the `privacy-policy` and `terms-of-service`
link types, as defined in
https://html.spec.whatwg.org/multipage/links.html#linkTypes, and in
RFC6903 before that.

### Description

This PR adds descriptions for the `privacy-policy` and `terms-of-service` link types, as defined in https://html.spec.whatwg.org/multipage/links.html#linkTypes.

### Motivation

Documenting things is good.

### Additional details

https://html.spec.whatwg.org/multipage/links.html#linkTypes
Sections 4 and 5 of RFC 6903
https://mikewest.github.io/privacy-policy-discovery/

### Related issues and pull requests

Depends on: https://github.com/whatwg/html/pull/9860
Depends on: https://github.com/whatwg/html/pull/9877

### CODE REVIEW

**Comment by estelle** [comment]
We can't add these to MDN until they are added to the specification. I'll convert to draft. Please change to ready for review when in the WHATWG specification. 

Alternatively, this can be closed, and then resubmit branch [mikewest:patch-1](https://github.com/mikewest/content/tree/patch-1) as a PR when ready for review, but will leave that decision up to you.

This PR says this is your first contribution to MDN, but I can't believe that! How are you? Long time no see.

**Comment by estelle** [comment]
@mikewest I had a discussion and read the contribution docs. We can actually add these to MDN now, but with edits. I'll let you decide how you want to proceed:

We can add now, with added language and/or macros citing that these are experimental, not supported, and not on any standards track or we can wait until these are added into WHATWG and then just list their support.

These two attribute values should also be added to browser compat - https://github.com/mdn/browser-compat-data/

**Comment by mikewest** [comment]
Hi, @estelle! There was a bit more back-and-forth on the PRs against HTML that I wanted to ensure resolved reasonably before merging this. The one has been merged, I expect the other to follow in the near future. Once they're in, I'll edit this PR accordingly, and ask for review.

Regarding browser compat data, I suspect no work is needed, as there's no web-facing behavior associated with these link types AFAIK.

**Comment by mikewest** [comment]
Both PRs were merged, so I'm putting this back up for review. I looked over the text again, but I don't think any of it changed.

File: files/en-us/web/html/attributes/rel/index.md [hunk 1]
```diff
@@ -41,9 +41,11 @@ The following table lists some of the most important existing keywords. Every ke
...
 | [`tag`](#tag)                                                        | Gives a tag (identified by the given address) that applies to the current document.                                                                                                                                                                                                     | Not allowed             | Link                                             | Not allowed             |
+| [`terms-of-service`](#terms-of-service)                              | Gives a link to a information about the information about the agreements between a document's provider and users who wish to use the document provided.                                                                                                                                 | Link                    | Link                                             | Not allowed             |
```
Comment (estelle) [inline files/en-us/web/html/attributes/rel/index.md:48]:
```suggestion
| [`terms-of-service`](#terms-of-service)                              | Link to the agreement, or terms of service, between the document's provider and users who wish to use the document.                                                                                                                                 | Link                    | Link                                             | Not allowed             |
```
alternatively, "Indicates that the linked resource is the terms of service for the current resource".

The description is linked below. This should be a quick description. See `help` as the example that should be used. 

This table has gotten too verbose. But that is an edit for a different PR.

**Review by estelle — COMMENTED** [review]
there was a grammar issue in the table. I suggested two shorter alternatives. Otherwise looks good,

I did notice a typo when reading the other entries as examples, so committed that fix in as I could not make a suggestion on that line.

**Comment by mikewest** [comment]
I'm happy with your suggestion and fix. Thanks for taking another look!

File: files/en-us/web/html/attributes/rel/index.md [hunk 2]
```diff
@@ -41,9 +41,11 @@ The following table lists some of the most important existing keywords. Every ke
...
 | [`tag`](#tag)                                                        | Gives a tag (identified by the given address) that applies to the current document.                                                                                                                                                                                                     | Not allowed             | Link                                             | Not allowed             |
+| [`terms-of-service`](#terms-of-service)                              | Link to the agreement, or terms of service, between the document's provider and users who wish to use the document.                                                                                                                                 | Link                    | Link                                             | Not allowed             |
```
[the remark this answers is not in the document]
Comment (mikewest) [reply files/en-us/web/html/attributes/rel/index.md:48]: It's not clear to me what the linter doesn't like in this line. It appears to be suggesting a no-op change?

**Review by estelle — APPROVED** [review]

**Pull request merged**